py_gtfs_rt_ingestion/ingest.py

```diff
@@ -9,11 +9,121 @@
 import pyarrow.parquet as pq
 
 from datetime import datetime
+from multiprocessing import Pool
+from multiprocessing import Queue
 from pathlib import Path
 
 from py_gtfs_rt_ingestion import Configuration
 
+
 DESCRIPTION = "Convert a json file into a parquet file. Used for testing."
+MULTIPROCESSING_POOL_SIZE = 4
+
+def gz_to_pyarrow(filename: str, config: Configuration, return_queue: Queue) -> None:
+    """
+    Accepts filename as string. Converts gzipped json -> pyarrow table. 
+
+    parrow table sent back to main process using return_queue.
+    If conversion fails, returns filename.
+    """
+    # Enclose entire function in try block, Process can't fail silently.
+    # Must return either pyarrow table or filename
+    try:
```

Comment:
Put all logic of function in `try` block. Multiprocess must return either pyarrow table or filename (if conversion failed).